src/app/components/plus-one/PlusOne.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component, PropTypes } from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import * as actions from './actions';
+
+class PlusOne extends Component {
+    constructor(props) {
+        super(props);
+
+        this.handleClick = this.handleClick.bind(this);
+    }
+
+    handleClick() {
+        const { increment } = this.props;
+        increment();
+    }
+
+    render() {
+        const { counter } = this.props;
+        return (
+            <p>
+                <button type="button" className="btn btn-primary" onClick={this.handleClick}>+ {counter}</button>
```

Comment:
What about
```js
<button
    type="button"
    className="btn btn-primary"
    onClick={this.handleClick}
>
    + {this.props.counter}
</button>
```
?

I got used to this format on Remy/Cafe App and I find it more readable. If you have only one prop, you keep it on one line, if you have more, you separate them by newlines.

But it’s just a proposal.